src/render/glyph_atlas.js

```diff
@@ -70,6 +71,9 @@ function makeGlyphAtlas(stacks: {[string]: {[number]: ?StyleGlyph}}): GlyphAtlas
     return {image, positions};
 }
 
-module.exports = {
+const exported = {
     makeGlyphAtlas
 };
+
+export default exported;
+export { makeGlyphAtlas };
```

Comment:
same question here as above (I assume it's intentional/related to default exports?)